src/CMakeLists.txt

```diff
@@ -205,6 +205,24 @@ if (GGML_CLBLAST)
     endif()
 endif()
 
+if (GGML_DNNL)
+    find_package(dnnl)
+    if (dnnl_FOUND)
+        message(STATUS "OneDNN found")
+
+        set(GGML_HEADERS_DNNL ggml-dnnl.h)
+        set(GGML_SOURCES_DNNL ggml-dnnl.cpp)
+
+        set(GGML_EXTRA_INCS  ${GGML_EXTRA_INCS}  ${CLBLAST_INC} ${OPENCL_INC})
```

Comment:
CLBLAST vars look out of place here

Reply:
Thank you - it was copy-pasted with mistake.
I've fixed it and some other parts of this file.